doc/api/cli.md

```diff
@@ -1385,6 +1385,20 @@ threadpool by setting the `'UV_THREADPOOL_SIZE'` environment variable to a value
 greater than `4` (its current default value). For more information, see the
 [libuv threadpool documentation][].
 
+## Useful v8 options
+
+These are v8 options and fall outside Node.js' responsibility.
+
+V8 options that are allowed are:
```

Comment:
I think this should either be removed completely or at least reworded. Node does not restrict the V8 options that can be passed on the command line.

Reply:
There is another place in the document mentioning the same thing.